tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -69,6 +72,71 @@ Test Metric "Active_Users" On OpenShift Monitoring On Cluster Monitoring Prometh
     Should Be Equal    ${value}    2    msg=Active Users are not equal to length of list or N users
     [Teardown]    CleanUp JupyterHub For N users    list_of_usernames=${list_of_usernames}
 
+Test Metric "Active Users" On Telemeter
+    [Documentation]    Verifies the openshift metrics and telemeter shows
+    ...                the same rhods active users
+    [Tags]    ODS-1054
+    ...       Tier1
+     @{list_of_usernames} =    Create List    ${TEST_USER_3.USERNAME}    ${TEST_USER_4.USERNAME}
+    Log In N Users To JupyterLab And Launch A Notebook For Each Of Them
+    ...    list_of_usernames=${list_of_usernames}
+    ${value} =    Run OpenShift Metrics Query    query=${METRIC_RHODS_ACTIVE_USERS}
+    ${cluster_id} =    Get Cluster ID
+    ${time} =    Get Start Time And End Time    interval=15m
+    ${steps} =    Set Variable    15
+    ${query} =    Set Variable    cluster:usage:consumption:rhods:active_users{_id=${cluster_id}}
+    ${url}=    Set Variable    ${telmeter_url}query_range?query=${query}&start=${time[0]}&end=${time[1]}&step=${steps}
+    Open Browser     ${url}    ${BROWSER.NAME}
```

Comment:
Telemeter is just a Grafana instance. Lines 89-91 could be replaced by this keyword at tests/Resources/Page/ODH/Grafana/Grafana.resource:

```
Launch Grafana    ocp_user_name=${OCP_ADMIN_USER.USERNAME}    ocp_user_pw=${OCP_ADMIN_USER.PASSWORD}    ocp_user_auth_type=${OCP_ADMIN_USER.AUTH_TYPE}    grafana_url=https://${grafana_url}   browser=${BROWSER.NAME}   browser_options=${BROWSER.OPTIONS}
```
And you could add a new keyword in Grafana.resource called `Run Range Query In Browser` similar to [this one](https://github.com/red-hat-data-services/ods-ci/blob/91e8d64dd18d6139801a13ccd88cf8c6dba9bc69/tests/Resources/Page/ODH/Prometheus/Prometheus.robot#L20)  but instead of using the RequestsLibrary.GET just do use the Go keyword to access it with the browser